po/extra/lt.po

```diff
@@ -0,0 +1,302 @@
+msgid ""
+"Create the shortcut to your favorite portable apps into your app launcher"
+msgstr ""
+"Pateikti greičiausią nuorodą iki mėgstamiausių perkeliamų programėlių tavo "
+"programų paleidiklyje"
+
+#: data/pinit.appdata.xml.in:11
+msgid ""
+"Create the shortcut to portable apps like raw executable files, AppImage "
+"files, etc. into the app launcher of your desktop environment."
+msgstr ""
+"Pateikti greičiausią nuorodą perkeliamoms programėlėms (kaip neapdorota "
+"vykdomoji rinkmena, ProgramėlėsVaizdinio rinkmena, t. t.) tiesiogiai "
+"programėlės paleidikliui tavo darbalaukio aplinkoje."
+
+#: data/pinit.appdata.xml.in:14
+msgid "Other features include:"
+msgstr "Kitos savybės įtraukia:"
+
+#: data/pinit.appdata.xml.in:18
+msgid "Edit or delete created app entries without opening the file manager"
+msgstr ""
+"Keisti arba trinti sukurtas programėlės įvestis neatidarant rinkmenų "
+"tvarkytuvės"
+
+#: data/pinit.appdata.xml.in:19
+msgid "Automatically add execution permission to the file you select"
+msgstr ""
+"Pagal nutylėjimą įtraukti vykdomąsias teises tavo pasirenkamoms rinkmenoms"
+
+#: data/pinit.appdata.xml.in:20
+msgid "Syntax error detection"
+msgstr "Sintaksės klaidos paieška"
+
+#: data/pinit.appdata.xml.in:21
+msgid ""
+"Automatically save everything―your data in editing, last open view, and "
+"preferences"
+msgstr ""
+"Pagal nutylėjimą viską išsaugoti ― tavo įvesties duomenis keitimo metu, "
+"paskutinio atidaryto rodinio metu, ir nustatymus"
+
+#: data/pinit.appdata.xml.in:26
+msgid "Files view in the light mode"
+msgstr "Rinkmenos rodinys šviesioje veiksenoje"
+
+#: data/pinit.appdata.xml.in:30
+msgid "Edit view in the light mode"
+msgstr "Keitimo rodinys šviesioje veiksenoje"
+
+#: data/pinit.appdata.xml.in:34
+msgid "Files view in the dark mode"
+msgstr "Rinkmenos rodinys tamsioje veiksenoje"
+
+#: data/pinit.appdata.xml.in:38
+msgid "Edit view in the dark mode"
+msgstr "Keitimo rodinys tamsioje veiksenoje"
+
+#: data/pinit.appdata.xml.in:48
+msgid "The biggest and greatest updates come to \"Pin It!\":"
+msgstr "Didžiausi ir geriausi atnaujinimai kartu su \"Pakišk tai!\":"
+
+#: data/pinit.appdata.xml.in:52
+msgid ""
+"Redesign to fit modern desktop environments, including support for "
+"responsive design"
+msgstr ""
+"Permodeliuota, kad tiktų naujoviškoms darbalaukio aplinkoms įskaitant ir "
+"grįžtamojo dizaino palaikymą"
+
+#: data/pinit.appdata.xml.in:53
+msgid "Migrate to GTK 4, the latest version of GTK"
+msgstr "Pokytis iki GTK 4 versijos, naujausia GTK versija"
+
+#: data/pinit.appdata.xml.in:55
+msgid "Other changes:"
+msgstr "Kiti pokyčiai:"
+
+#: data/pinit.appdata.xml.in:59
+msgid "Support setting \".ico\" files as the app icon"
+msgstr "Įgyvendintas \".ico\" rinkmenų nustatymas kaip programėlės ikonėlė"
+
+#: data/pinit.appdata.xml.in:60
+msgid "Support setting \"StartupWMClass\" key"
+msgstr "Įgyvendintas \"WM klasė paleidimo metu\" nustatymo identifikatorius"
```

Comment:
[Changes request]
The word `StartupWMClass` is a term in the desktop file specification actually, so we should keep it as it is. (I should have left a comment here too, sorry……)

Reply:
Yeah, i had resolved this in po/extra/pl.po 
I thought so that You gonna Hardcode this term in Your app.
No need to apologize, I were too lazy for the language flush in po/extra/lt.po so sorry.

Reply:
Yeah it would be great if we could hardcode what are not intended to be translated but I'm not sure if XML support that. At least I'll keep the current source text as it is until your pull request is merged.

Reply:
Let us roll back to the word which You are requesting.